consoletest_finish-gnome-TW-20170721.json

```diff
@@ -0,0 +1,27 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "xpos": 0,
+      "width": 1024,
+      "ypos": 0,
+      "height": 633,
+      "type": "match"
+    },
+    {
+      "width": 152,
+      "xpos": 433,
+      "ypos": 0,
+      "height": 42,
+      "type": "exclude"
+    }
+  ],
+  "tags": [
+    "x11",
+    "generic-desktop",
+    "reboot-after-installation",
+    "ENV-DESKTOP-gnome",
+    "ENV-DISTRI-opensuse",
+    "ENV-INSTLANG-en_US"
+  ]
+}
```

Comment:
So there were 2 needles which do the job, I've removed old one and introduced new one with update are excluded, as well as time.